app/src/main/java/org/p2p/wallet/history/model/TransactionDetailsLaunchState.kt

```diff
@@ -5,8 +5,9 @@ import kotlinx.parcelize.Parcelize
 
 sealed class TransactionDetailsLaunchState : Parcelable {
     @Parcelize
-    data class Id(val transactionId: String) : TransactionDetailsLaunchState()
+    data class Id(val tokenPublicKey: String, val transactionId: String) : TransactionDetailsLaunchState()
 
```

Comment:
it's better to keep formatting as one for all the descendants 

```suggestion
    data class Id(val tokenPublicKey: String, val transactionId: String) : 
        TransactionDetailsLaunchState()

```